spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/write/context/BigQueryDirectDataSourceWriterContext.java

```diff
@@ -80,6 +81,7 @@ enum WritingMode {
   }
 
   private WritingMode writingMode = WritingMode.ALL_ELSE;
+  private final Optional<ImmutableList<String>> clusteredFields;
```

Comment:
Any reason for clusteredFields to be a member? they are used once in `getOrCreateTable()` (line 189)  and this is being called from the constructor. It makes this call sensitive to member initialization.

Reply:
Yeah we don't need it as a member, fixed.